src/Tribe/Main.php

```diff
@@ -452,6 +452,9 @@ public function bind_implementations(  ) {
 			// Gutenberg Extension
 			tribe_singleton( 'tec.gutenberg', 'Tribe__Events__Gutenberg', array( 'hook' ) );
 
+			// Admin Notice for Timezones
+			tribe_singleton( 'tec.admin.notice.timezones', 'Tribe__Events__Admin__Notice__Timezones' );
```

Comment:
When registering the singleton we can pass a third param for a method to be called when initially loading the class.

```
tribe_singleton( 'tec.admin.notice.timezones', 'Tribe__Events__Admin__Notice__Timezones', array( 'hook' ) );
```